core/templates/rid_owner.h

```diff
@@ -193,7 +193,7 @@ class RID_Alloc : public RID_AllocBase {
 			if (THREAD_SAFE) {
 				spin_lock.unlock();
 			}
-			if (validator_chunks[idx_chunk][idx_element] & 0x80000000) {
+			if ((validator_chunks[idx_chunk][idx_element] & 0x80000000) && validator_chunks[idx_chunk][idx_element] != 0xFFFFFFFF) {
```

Comment:
I also added this small fix, which corrects the error (said it was uninitialized when it was actually freed)

Reply:
FYI, there was this fix a week ago which also impacted this: https://github.com/godotengine/godot/commit/37db8d61845f1bf22d50939268aa8baa2663d06b#diff-aaa31181a1479fa3b274adc649b7bb6b3ac392c8bd2a623b39c3b88810b12051